lib/sorbet-rails/railtie.rb

```diff
@@ -34,6 +34,7 @@ def inherited(child)
             child.send(:public_constant, :ActiveRecord_Relation)
             child.send(:public_constant, :ActiveRecord_AssociationRelation)
             child.send(:public_constant, :ActiveRecord_Associations_CollectionProxy)
+            child.send(:public_constant, :RelationType) if child.const_defined?(:RelationType)
```

Comment:
This constant doesn't exist and we'll have to define it for the child.
Something like 
```
# define the type alias dynamically
# if we are unable to use ActiveRecord_Relation, we would have to get it manually
# active_record_relation = const_get(:ActiveRecord_Relation)
relation_type = T.type_alias { 
  ActiveRecord_Relation, 
  ActiveRecord_AssociationRelation, 
  ActiveRecord_Associations_CollectionProxy
}
# create a constant mapping to the type alias.
child.const_set(:RelationType, relation_type)
```

Reply:
ok, I have to admit that this file was a bit beyond my understanding... 😅